meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,135 @@
+# v0.29.0 release changelogs
+
+In this release, we integrate a demanded feature: the possibility to require that all query terms are present in search results. Some users have been stuck in the pre-0.21 versions due to that.
+We also made a lot of improvements regarding the indexing speed!
```

Comment:
```suggestion
In this release, we integrated a demanded feature: all query terms being present in search results. Some users have been stuck in the pre-0.21 versions because of this.
 We also improved the indexing speed!
```

Reply:
```suggestion
In this release, we integrated a demanded feature: all query terms being present in search results. Some users have been stuck in the pre-0.21 versions because of this.
We also improved the indexing speed
```